Gordon360/Static Classes/Names.cs

```diff
@@ -110,11 +110,17 @@ public static class SQLQuery
     {
         public static string ALL_PUBLIC_STUDENT_REQUEST = "SELECT ISNULL(Mail_Location, '') as Mail_Location, ISNULL(BuildingDescription, '') as Hall, ISNULL(FirstName, '') as FirstName, ISNULL(LastName, '') as LastName, ISNULL(NickName, '') as NickName, ISNULL(Class, '') as Class, ISNULL(Major1Description, '') as Major1Description, ISNULL(Major2Description, '') as Major2Description, ISNULL(Major3Description, '') as Major3Description, ISNULL(Minor1Description, '') as Minor1Description, ISNULL(Minor2Description, '') as Minor2Description, ISNULL(Minor3Description, '') as Minor3Description, ISNULL(HomeCity, '') as HomeCity, ISNULL(HomeState, '') as HomeState, ISNULL(Country, '') as Country, ISNULL(KeepPrivate, '') as KeepPrivate, ISNULL(Email, '') as Email, AD_Username FROM STUDENT S WHERE AD_Username is not null FOR JSON PATH, ROOT('Students')";
         public static string ALL_PUBLIC_FACULTY_STAFF_REQUEST = "SELECT FirstName, LastName, NickName, OnCampusDepartment, BuildingDescription, HomeCity, HomeState, Country, KeepPrivate, JobTitle, Email, Type, AD_Username, Mail_Location FROM FacStaff WHERE AD_Username is not null";
-        public static string ALL_PUBLIC_ALUMNI_REQUEST = "SELECT FirstName, LastName, NickName, Major1Description, Major2Description, HomeCity, HomeState, Country, Email, ShareName, PreferredClassYear, AD_Username FROM Alumni WHERE ShareName is null OR ShareName = 'Y';";
+        //ALL_PUBLIC_ALUMNI_REQUEST returns search result to the advanced people search 
+        //with encrypted id as the ad_username if their ad_username is null
+        //ALL_PUBLIC_ALUMNI_REQUEST and ALL_ALUMNI_REQUEST queries work together
+        public static string ALL_PUBLIC_ALUMNI_REQUEST = "SELECT FirstName, LastName, NickName, Major1Description, Major2Description, HomeCity, HomeState, Country, Email, ShareName, PreferredClassYear, ISNULL(AD_Username, CONVERT(NVARCHAR(32),HashBytes('MD2', ID),2)) as AD_Username FROM Alumni WHERE ShareName is null OR ShareName = 'Y';";
 
         public static string ALL_STUDENT_REQUEST = "SELECT * from Student WHERE AD_Username is not null";
         public static string ALL_FACULTY_STAFF_REQUEST = "SELECT * from FacStaff WHERE AD_Username is not null";
-        public static string ALL_ALUMNI_REQUEST = "SELECT * from Alumni WHERE AD_Username is not null";
+        //ALL_ALUMNI_REQUEST returns the profile information for the first match of 
+        //either real ad_username (if exists) or the encrypted id
+        //from the advanced people search to the encrypted ad_username of their profile 
```

Comment:
This comment is incorrect and misleading. The `ALL_ALUMNI_REQUEST` query returns the set of all the alumni in our records. Other methods query this set to find a specific alum based on AD_Username (or the encrypted id now), but this itself is all of them.